src/vs/workbench/parts/tasks/common/problemMatcher.ts

```diff
@@ -1000,11 +1050,12 @@ export namespace Schemas {
 
 	export const NamedProblemPattern: IJSONSchema = Objects.deepClone(ProblemPattern);
 	NamedProblemPattern.properties = Objects.deepClone(NamedProblemPattern.properties);
```

Comment:
I would code this different since we don't want to loose to set the name if the clones
properties are undefined | null. Something like

NamedProblemPattern.properties = Objects.deepClone(NamedProblemPattern.properties) || {};

Then the if can be removed.